### PR TITLE
[RFR] Improved SCSS code consistency

### PR DESCRIPTION
**Pull request by HugoGiraudel**
Cheers. :)

### CODE REVIEW

File: source/scss/1977.scss [hunk 1]
```diff
@@ -10,7 +10,7 @@
   filter: contrast(1.1) brightness(1.1) saturate(1.3);
 
   &:after{
-    background: rgba(243, 106, 188, .3);
+    background: rgba(243, 106, 188, 0.3);
```
Comment (una) [inline source/scss/1977.scss]: I generally omit leading 0's as a rule :smile:

File: source/scss/aden.scss [hunk 2]
```diff
@@ -9,10 +9,10 @@
 %aden,
 .aden {
   @extend %filter-base;
-  filter: hue-rotate(-20deg) contrast(.9) saturate(.85) brightness(1.2);
+  filter: hue-rotate(-20deg) contrast(0.9) saturate(0.85) brightness(1.2);
 
-  &:after{
+  &::after {
```
Comment (una) [inline source/scss/aden.scss]: What's the reason for adding a second `:`?

File: source/scss/1977.scss (hunk 1 above)
Comment (HugoGiraudel) [inline source/scss/1977.scss]: Oh. You did not do it everywhere, so I just assumed it was better to add it.

My 2 cents on this is that it is the minifier’s role to shorten your code, not yours. :)

File: source/scss/aden.scss (hunk 2 above)
Comment (HugoGiraudel) [inline source/scss/aden.scss]: Sticking to the spec. :)

File: source/scss/1977.scss (hunk 1 above)
Comment (una) [inline source/scss/1977.scss]: Ahh! I intended to remove them if I have any leading 0's. Thank you for the changes! If you take the leading 0's back out, I'd be happy to merge! :+1:

**Comment by HugoGiraudel** [comment]
Done.

**Comment by una** [comment]
Perfect :) Please pull changes so it's mergable

**Pull request merged**